### ISSUE TITLE
Task appears as 'unnamed' in control center

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#490]
```
What steps will reproduce the problem?
1. Create a workflow (see attachment)
2. Declare a local net variable a of type string
3. Create a task A using decompose to direct data transfer
4. Use variable a as input and output
5. Load the workflow into engine and start it


What is the expected output? What do you see instead?
I would expect that the task name 'A' appears in the worklist.
Instead the work item appears as 'unnamed'

What version of the product are you using? On what operating system?
YAWL 2.3.5 with two recent classes.

Please provide any additional information below.
Using Linux OS.
The problem occured sporadically in many workflows recently.

```

Original issue reported on code.google.com by `andreas....@gmail.com` on 3 Dec 2013 at 1:17

Attachments:
* [wf.yawl](https://storage.googleapis.com/google-code-attachments/yawl/issue-490/comment-0/wf.yawl)

**Comment by yawlfoundation** [comment]
```
This is a known issue caused by a bug in the v2.3 and later editor, and is 
resolved in editor v3, available in the next couple of days 
```

Original comment by `yawl.mic...@gmail.com` on 3 Dec 2013 at 1:26
* Changed state: **Accepted**

**Comment by yawlfoundation** [comment]
```
resolved in editor 3.0
```

Original comment by `yawl.mic...@gmail.com` on 9 Dec 2013 at 2:47
* Changed state: **Fixed**
